script.js

```diff
@@ -1,4 +1,138 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+//Create game mode for Player 1
+var playerMode = "Player 1";
+
+//Create function for random dice roll
+var rollDice = function () {
+  // return 2;
+  var randomFloat = Math.random() * 6;
+  var resultInteger = Math.floor(randomFloat) + 1;
+  return resultInteger;
+};
+
+//Create arrays to hold player's dice rolls
+var diceRollArray = function () {
+  var generateDiceRoll = [rollDice(), rollDice()];
+  if (playerMode === "Player 1") {
+    player1DiceRoll = generateDiceRoll;
+  } else {
+    player2DiceRoll = generateDiceRoll;
+  }
+  return generateDiceRoll;
+};
+
+//Generate a function to combine 2 strings in the array for player 1
+var numPlayer1DiceArray = function (playerInput) {
+  var player1Array0 = String(player1DiceRoll[0]);
+  var player1Array1 = String(player1DiceRoll[1]);
+  if (playerInput == 1) {
+    player1CombineString = player1Array0 + player1Array1;
+  } else {
+    player1CombineString = player1Array1 + player1Array0;
+  }
+  return player1CombineString;
+};
+
+//Generate a function to combine 2 strings in the array for player 2
+var numPlayer2DiceArray = function (playerInput) {
+  var player2Array0 = String(player2DiceRoll[0]);
+  var player2Array1 = String(player2DiceRoll[1]);
+  if (playerInput == 1) {
+    player2CombineString = player2Array0 + player2Array1;
+  } else {
+    player2CombineString = player2Array1 + player2Array0;
+  }
+  return player2CombineString;
+};
```

Comment:
Good job in using helper functions and in using a string to concatenate the numbers! Would it be possible to combine the two helper functions since both achieve the same outcome but for different players and use the playerMode to determine which player's arrays it needs to use?